README.md

```diff
@@ -61,6 +39,11 @@ You'll find all of the documentation for this SDK on [our Mapbox Navigation page
 
 ## Using Snapshots
 
+#### `1.0.0`+ versions of the Navigation SDK:
+
+TODO: FILL IN
```

Comment:
Currently Downloads API doesn't have `SNAPSHOT`s support so let's leave this `TODO` for now until that's resolved cc @abhishek1508

Reply:
Ok, I'll add some sort of "Coming soon..." sentence or two.